### DISCUSSION TITLE
CNAME DNS on different domain

### DISCUSSION THREAD

**Discussion by frederikhors** [caprover/caprover#1895] [Q&A] [answered]
I'm trying to understand if I can use the actual caprover configuration to accomplish this:

- I have the `domain-1.com` with this DNS setting:
    - A | app.domain-1.com | 123.456.123.456 (this is a fake IP here)

- I would like to have a second domain `domain-2.com` with this DNS setting:
    - CNAME | app.domain-2.com | app.domain-1.com

In caprover HTTP Settings I have:

- `app.domain-1.com` with HTTPS enabled (because I have the `A` DNS and IP)

- `app.domain-2.com` without HTTPS (because Let's encrypt needs `A` DNS)

- checked __Force HTTPS by redirecting all HTTP traffic to HTTPS__

It works if I visit `app.domain-1.com`.

I'm getting the `Nothing here yet :/` blue page if I visit `app.domain-2.com` (both using http:// and https://).

Why?

I would expect to see the same page of `app.domain-1.com` if I visit `app.domain-2.com`.

Am I wrong?

**Comment by githubsaturn** [comment, marked as answer]
You're right. Most likely you have a typo or something... try restarting your server to see if it works
